provider/datagen/src/transform/cldr/time_zones/convert.rs

```diff
@@ -119,45 +119,48 @@ impl Location {
 
 impl From<CldrTimeZonesData<'_>> for ExemplarCitiesV1<'static> {
     fn from(other: CldrTimeZonesData<'_>) -> Self {
-        let time_zone_names_data = other.time_zone_names_resource;
-        let bcp47_tzid_data = &compute_bcp47_tzids_hashmap(other.bcp47_tzids_resource);
         Self(
-            time_zone_names_data
-                .zone
-                .0
+            other
+                .bcp47_tzids_resource
                 .iter()
-                .flat_map(|(key, region)| {
-                    region
-                        .0
-                        .iter()
-                        .flat_map(move |(inner_key, place_or_region)| {
-                            let mut key = key.clone();
-                            key.push('/');
-                            key.push_str(inner_key);
-                            match place_or_region {
-                                LocationOrSubRegion::Location(place) => {
-                                    match bcp47_tzid_data.get(&key) {
-                                        Some(bcp47) => place
-                                            .exemplar_city()
-                                            .map(|city| vec![(bcp47, city)])
-                                            .unwrap_or_default(),
-                                        None => panic!("Cannot find bcp47 for {key:?}."),
+                .filter_map(|(bcp47, bcp47_tzid_data)| {
+                    bcp47_tzid_data
+                        .alias
+                        .as_ref()
+                        .map(|aliases| (bcp47, aliases))
+                })
+                .filter_map(|(bcp47, aliases)| {
+                    aliases
```

Comment:
Note: There are 2 definitions of the canonical IANA alias for each time zone; one is the CLDR alias, which is more historical and always stable, and the other is the IANA alias, which tends to be more modern but is subject to change. The order of the aliases in the bcp47 file does not seem to correspond to whether it is the CLDR alias or the IANA alias. I know there are open issues about this, as well as an ECMA-402 proposal in the works to settle on whose alias is the one the industry should use.

Reply:
There's https://unicode-org.atlassian.net/browse/CLDR-14453